packages/@n8n/db/package.json

```diff
@@ -21,6 +21,12 @@
     "dist/**/*"
   ],
   "dependencies": {
+    "@n8n/config": "workspace:^",
+    "@n8n/di": "workspace:^",
+    "@n8n/typeorm": "catalog:",
+    "n8n-core": "workspace:^",
```

Comment:
we only pull `InstanceType` and `Class` from core. maybe those can be moved to the workflow package to simplify the dependency tree a bit.

Reply:
From a quick check, some migrations and repositories also depend on core and will be moved to `@n8n/db`, so I'd say let's aim to do this after they're moved over and we have full view of everything that needs to move out of core.